plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#
+# check the certificate validity of your certfificates
+#
+# andreas perhab - andreas.perhab@wt-io-it.at
+# https://www.wt-io-it.at/
+#
+# License: GPLv2
+#
+# Installing: Add list of your certificates prefixed by the type in munin plugin-conf.d
+#
+# For openvpn ca.crt and crl.pem
+# [cert_validity]
+# user root
+# env.CERTS crl:/etc/openvpn/easy-rsa/keys/crl.pem x509:/etc/openvpn/easy-rsa/keys/ca.crt
+#
+# For letsencrypt certficates
+# [cert_validity]
+# user root
+# env.CERTS x509:/etc/letsencrypt/live/domain1.example.com/cert.pem x509:/etc/letsencrypt/live/domain2.example.com/cert.pem
+#
+# Dependencies: openssl
+#
+# Parameters:
+#
+#  config
+
+
+if [ "$1" == "config" ] ; then
+	echo "graph_title Certificate validity"
+	echo "graph_args --logarithmic --base 1000"
+	echo "graph_vlabel certificate validity in days"
+	echo "graph_category certificates"
+fi
+
+now=$(date +%s)
+for cert in ${CERTS[@]}; do
```

Comment:
In case of `sh` (instead of `bash`), `$CERTS` should provide the same result.

Reply:
applied suggestions for sh instead of bash (could not test sh compatiblity on our servers because /bin/sh is symlinked in debian and ubuntu)

Reply:
This should be fine. At least my understanding of `/bin/sh` is, that it is just *the default shell of the system*. Quite recently Debian (and thus Ubuntu) switched from `/bin/sh` being provided by `/bin/bash` to `/bin/dash`. Thus there is no single one implementation. `shellcheck -s dash` is the base reference, we usually use for munin plugins in this repository.